src/kademlia/rpc_manager.cpp

```diff
@@ -158,7 +159,7 @@ void observer::set_id(node_id const& id)
 enum { observer_size = max3<
 	sizeof(find_data_observer)
 	, sizeof(announce_observer)
-	, sizeof(null_observer)
+	, sizeof(put_data_observer)
```

Comment:
ok, then actually we need get_item_observer, get_peers_observer, traversal_observer as well? 

Actually I guess the size should be same, only put_data_observer is bigger than others since it keep a string for token. 

But generally, we should put all observers here I think.